capstone-backend/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -61,22 +68,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         String profileImageUrl = (String) payload.get("picture");
 
         googleUser = new User(googleId, email, fullname, profileImageUrl,
-            new ImmutableMap.Builder<String, String>()
-              .put("token_id", tokenId)
-              .put("token_type", token_type)
-              .put("access_token", access_token)
-              .put("scope", scope)
-              .put("idpId", idpId)
-              .build());
+            new ImmutableMap.Builder<String, String>().put("token_id", tokenId).put("token_type", token_type)
+                .put("access_token", access_token).put("scope", scope).put("idpId", idpId).build());
 
       } else {
         System.err.println("Invalid ID token.");
         response.sendError(HttpServletResponse.SC_UNAUTHORIZED);
         return;
       }
 
-      // TODO(#47): Update Acess Token if it expires.
-
```

Comment:
I don't see any logic in this PR to update the Access Token. Did I miss it or was it in another PR?

Reply:
My UserServlet doPost updates the User object on Firestore. If the token does expire and change Firestore will update the store User.